src/components/table/Table.vue

```diff
@@ -126,7 +126,7 @@
                         <!-- Do not add `key` here (breaks details) -->
                         <!-- eslint-disable-next-line -->
                         <tr
-                            v-if="detailed && isVisibleDetailRow(row)"
+                            v-if="detailed && !customDetailRow && isVisibleDetailRow(row)"
```

Comment:
I've to suggest use a method instead, just to avoid more than 2 conditions on an if statement. An name for this method could be something like `isVisibleWhenDetailed(row)`